scripts/collector/mysql/sql/data_types.sql

```diff
@@ -16,12 +16,12 @@ with src as (
         i.TABLE_NAME,
         i.DATA_TYPE
 )
-select concat(char(39), @DMA_MANUAL_ID, char(39)) as PKEY,
-    concat(char(39), @DMA_SOURCE_ID, char(39)) as DMA_SOURCE_ID,
-    concat(char(39), @DMA_MANUAL_ID, char(39)) as DMA_MANUAL_ID,
-    src.table_catalog,
-    src.table_schema,
-    src.table_name,
-    src.data_type,
-    src.data_type_count
+select concat(char(39), @PKEY, char(39)) as pkey,
```

Comment:
maybe char(34) `"` instead of char(39) `'` everywhere?
since mssql also has PerfMonData with quotes ""